doc/neo-tree.txt

```diff
@@ -198,6 +198,13 @@ s             = open_vsplit: Same as open, but opens in a vertical split.
 
 t             = open_tabnew: Same as open, but opens in a new tab.
 
+                  open_drop: Same as open, but opens with the `:drop` command. 
+                             (run `:help drop` for more info)
+
+              open_tab_drop: Same as open, but opens in a new tab with the 
+                             `:tab drop` command. (run `:help drop` for more
```

Comment:
Same as above, this can be changed to
```suggestion
                             `:tab drop` command. (see |:drop| for more
```

Reply:
https://github.com/rkoval/neo-tree.nvim/commit/82cd313328ebfaf42409611a739d1ba5f8467173